test/BodyParams/BodyParamsMiddlewareTest.php

```diff
@@ -8,6 +8,7 @@
 namespace ZendTest\Expressive\Helper\BodyParams;
 
 use PHPUnit_Framework_TestCase as TestCase;
+use Psr\Http\Message\ResponseInterface;
```

Comment:
I don't think that the use of this interface to provide better IDE hint is sufficient to import its namespace here.